### ISSUE TITLE
"String can't be coerced into Float" error when using relative codespan font size

### ISSUE THREAD

**Issue by tchjunky** [asciidoctor/asciidoctor-pdf#2281]
### Issue

Following the Asciidoctor PDF documentation here: https://docs.asciidoctor.org/pdf-converter/latest/theme/codespan/#font-size

Applying monospace style operators to table cells and table columns results in an asciidoctor-pdf error when the pdf theme.yml file uses relative "em" font sizes for codespan. 

**my-theme.yml**
```
codespan:
  font-size: 8.5em
```

**table.adoc**
```
.Table with AsciiDoc Formatted Cell
[cols=",,m",options="header"]
|===
|Column 1
|Column 2
|Column 3

m|Row 1 left
|Row 1 middle
|Row 1 right

m|Row 2 left
a|AsciiDoc formatted cell

* List bullet
* List bullet

m|Row 2 right

m|Row 3 left
|Row 2 middle
|Row 2 right
|===
```

**Expected result**

Highlighted cells in the image below are formatted following the custom codespan font size (screenshot from VSCode preview).

![image](https://user-images.githubusercontent.com/4414166/178579917-6556207d-49e7-4929-8a78-03d6e8803ab4.png)

**Actual result**

asciidoctor-pdf outputs the following error and a PDF is not generated.

```
$ asciidoctor-pdf -a pdf-theme=my-theme.yml table.adoc
String can't be coerced into Float
  Use --trace to show backtrace
```

If `font-size` is changed from `8.5em` to `10`, asciidoctor-pdf generates the PDF below.

![image](https://user-images.githubusercontent.com/4414166/178580800-7d223cb3-60b9-4b16-aecf-d641574a9f56.png)

**Affected versions**

* Asciidoctor PDF 2.0.8 using Asciidoctor 2.0.17
* Asciidoctor PDF 2.1.5 using Asciidoctor 2.0.17

Please let me know if you need any additional information. 

Thanks!

**Comment by mojavelinux** [comment]
Wow, I don't know how we missed that. Thanks for bringing this to my attention. I'll get it patched up and included in the next release.

**Comment by mojavelinux** [comment]
resolved by f859909ae0172d445ff758eb623be6e2219f957e
